### ISSUE TITLE
Regarding Inferencing.

### ISSUE THREAD

**Issue by BakingBrains** [dqshuai/MetaFormer#9]
I have trained the model for 28 epochs on CUB-200 dataset, also I wrote a small inferencing script which accepts a single image along with its meta info. but while predicting it is not at all giving good result. Do I need to train more?

Any suggestions here?

Thank you.

**Comment by dqshuai** [comment]
I recommend training more epochs, like 100 or 200. If you use a pre-trained model, it is recommended to train with a smaller learning rate (1e-4, 1e-5).

**Comment by BakingBrains** [comment]
@dqshuai Thank you. Will train more with the mentioned LR.
